src/mgos_sntp.c

```diff
@@ -108,7 +108,7 @@ static bool mgos_sntp_query(const char *server) {
 static void mgos_sntp_retry_timer_cb(void *user_data) {
   s_state.retry_timer_id = MGOS_INVALID_TIMER_ID;
   const char *server = mgos_sys_config_get_sntp_server();
-  if (s_state.idx == 0 && s_state.local_server[0] != '\0') {
+  if (s_state.idx == 0 && s_state.local_server[0] != '\0' && mgos_sys_config_get_sntp_follow_dhcp_ntp()) {
```

Comment:
please run `clang-format` on changed files.

Reply:
mate, that line is actually correct even with clang-format run on it. I've even added clang-format to the folder with another commit.

Reply:
thanks for adding .clang-format.
it can't be ok though, google style specifies 80 column limit and this line is 100+

Reply:
CLion is saying it's fine.

Reply:
wait a second, updating the lines manually and the config fetch to match new name

Reply:
it's not though, either it's not using the `.clang-format` or something else is amiss.
ok, just break the line at `&&`, that's all that's needed anyway.